schemas/FeatureManagement.Dotnet.v1.0.0.schema.json

```diff
@@ -0,0 +1,91 @@
+{
+  "definitions": {},
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "title": ".NET Feature Management Configuration",
+  "required": [],
+  "patternProperties": {
+    "^[^:]*$": {
+      "description": "Declares a feature flag.",
+      "oneOf": [
+        {
+          "type": "boolean"
+        },
+        {
+          "type": "object",
+          "required": [
```

Comment:
I was looking at `ConfigurationFeatureDefinitionProvider` and I saw [this comment](https://github.com/microsoft/FeatureManagement-Dotnet/blob/d19c9ad045968099cca87cea024ff08ad3744307/src/Microsoft.FeatureManagement/ConfigurationFeatureDefinitionProvider.cs#L207) about how we support strings as the value. If so, then maybe that needs to be included in the `oneOf` section. If it isn't supported maybe we could remove those couple lines of the comment?

Reply:
```
myDisabledFeature3: "",
```
I think we don't really like users to declare a feature flag in this way, but under our current implementation, even if the user writes it like this, we will parse this feature flag as always disabled. 

> If it isn't supported maybe we could remove those couple lines of the comment?

I agree. I will make some updates to the comment.